client/core/core.go

```diff
@@ -142,6 +142,12 @@ type dexConnection struct {
 	// trades tracks outstanding orders issued by this client.
 	trades map[order.OrderID]*trackedTrade
 
+	failedTradesMtx sync.RWMutex
+	// failedTrades tracks trades that were unable to load at startup.
+	// Attempts to resume these trades are made when wallets are unlocked.
+	// Trades that are successfully resumed will be removed from this slice.
+	failedTrades []*trackedTrade
```

Comment:
Can we do this differently? If my read is right, these orders wouldn't show up in the e.g. `User` until they connect their wallet. I'd really like all orders to make it in to the `dexConnection.trades` map ASAP so that the user can at least see them.

Reply:
Yeah the behavior is unchanged for the user, except instead of having to re-login after unlocking the wallets, it will automatically load the trades when the wallets it needs are unlocked. I thought the whole point of the logic currently in `loadDBTrades` (which I have now moved to `checkIfTradesAreReady`) was to avoid having trades that have wallets which are unable to be unlocked to be in `dexConnection.trades`, so that `tick` is not constantly called with them.

I was thinking to use the `failedTrades` slice in another PR to create a popup after the user logs in to notify them that some trades could not be loaded due to whichever asset's wallet not being connected. Also, the "Your Orders" section could have some sort of red button saying "Active Trades failed to Load" that allows the user to see the popup again.